src/server/chromium.ts

```diff
@@ -191,6 +192,11 @@ export class Chromium extends AbstractBrowserType<CRBrowser> {
       chromeArguments.push('--no-startup-window');
     return chromeArguments;
   }
+
+  private _devtools(): CRDevTools | undefined {
+    if (getFromENV('PLAYWRIGHT_DEBUG_UI'))
```

Comment:
Maybe `PLAYWRIGHT_DEBUG`? We might want to guard other debugging features behind this env.